src/server/game/Instances/InstanceScript.cpp

```diff
@@ -306,6 +307,19 @@ bool InstanceScript::SetBossState(uint32 id, EncounterState state)
                         if (minion->isWorldBoss() && minion->IsAlive())
                             return false;
 
+            switch (state)
+            {
+                case IN_PROGRESS:
+                    SendEncounterStart(1, 0, 1080000, 1080000); // Same values in all packets
```

Comment:
The timer isn't constant - depends on number of players in raid (scales between 10 and 30), obviously constant for 5 man dungeons